fluidreview/api.py

```diff
@@ -139,3 +150,45 @@ def request(self, method, url_suffix, **kwargs):
             self.session = self.initialize_session()
             response = getattr(self.session, method)(urljoin(BASE_API_URL, url_suffix), **kwargs)
         return response
+
+
+def process_user(fluid_user):
+    """
+    Create/update User and Profile model objects based on FluidReview user info
+
+    Args:
+        fluid_user (ReturnDict): Data from a fluidreview.serializers.UserSerializer object
+    """
+    user = User.objects.filter(email=fluid_user['email']).first()
+    if not user:
+        user = User.objects.create(email=fluid_user['email'], username=fluid_user['email'])
```

Comment:
Can we use `get_or_create` here like you do in the next line?

Reply:
It's possible that there might be an existing user with that email but a username that doesn't equal the email (if someone logged in before a fluidreview import). In that case, a get_or_create with email and username will create a new user.  A get_or_create with email only will also require assigning the username and saving if new/created.  So I decided to do a filter on email followed by a create with email and username if nothing is returned.

Reply:
I think you can specify `username` in `defaults`, like this:

    user, _ = User.objects.get_or_create(email=fluid_user['email'], defaults={'username': fluid_user['email']})

Reply:
will do, thanks!